app/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "securedrop",
+  "productName": "SecureDrop",
+  "version": "0.0.1",
+  "description": "",
+  "main": ".vite/build/main.js",
+  "scripts": {
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "make": "electron-forge make",
+    "publish": "electron-forge publish",
+    "lint": "eslint --ext .ts,.tsx ."
+  },
+  "keywords": [],
+  "author": {
+    "name": "SecureDrop Team",
+    "email": "securedrop@freedom.press"
+  },
+  "license": "MIT",
```

Comment:
This should be AGPL, although I'm not sure how much it matters what we specify here unless we publish the package itself.